project/core/templates/landing.html

```diff
@@ -1,172 +1,183 @@
+                            The structure both allows a community of volunteers to collaborate on a single political issue,
+                            and reserves space for dissenting opinions.
+                        </div><br><br>
+                    </td>
+                </tr>
+                <tr class="tr-why">
+                    <td><h2 class="subtitle">Personalized Policy Feed</h2>
+                        <div class="text"> CiviWiki intelligently personalizes users' feed in two meaningful ways.
+                        <br>
+                        <br> First, the issues promoted to users' feed will be personalized to the user's expressed
+                        interests, and the timeliness of the issue.
+                        <br>
+                        <br> Second, the structure of the issue topics break policy positions into bite-sized
+                        contentions we call Civies. Each Civi is logically related to the rest of the topic. Based on
+                        the user's support, opposition, or neutrality to each Civi, CiviWiki promotes different relevant
+                        content.</div><br><br>
+                    </td>
+                    <td>
+                        <span class="icon">
+                            <i class="large material-icons">web</i>
+                        </span>
+                    </td>
+                </tr>
+                <tr class="tr-why">
+                    <td><span class="icon"><i class="large material-icons">location_city</i></span></td>
+                    <td><h2 class="subtitle">Citizen/Representative Engagement</h2>
+                        <div class="text">CiviWiki's core goal is to engage citizens and their representatives, with the goal
+                        of making government more accountable.
+                        <br>
+                        <br> First, CiviWiki will organize the user's policy profile and compare it to every political
+                        candidate in the user's district. This quick, detailed, comparison will help users make informed
+                        votes, and we believe increased voter confidence will increase voter turnout.
+                        <br>
+                        <br> Second, CiviWiki will collect anonymized user data and forward district-level statistics to
+                        representatives. With a critical mass of users, we believe timely district level polling data
+                        will influence representatives votes.</div><br><br></td>
+                </tr>
+            </table><br><br>
+        </div><br><br><br>
+        <div class="subtitle center">
+            <p>See what your voice can bring to your community!</p><br><br><a href="/login/"><button class="ripple">Join CiviWiki</button></a>
+        </div><br><br><br><br>
+        <div class="subtitle center">
+            <p class="text">CiviWiki is still in development!</p><br><br>
```

Comment:
Add translate, remove line breaks.

```suggestion
            <p class="text">{% translate "CiviWiki is still in development!" %}</p>
```